src/test/parse-fail/qquote-1.rs

```diff
@@ -10,61 +10,51 @@
 
 // compile-flags: -Z parse-only
 
-// ignore-test Can't use syntax crate here
+// ignore-cross-compile
 
-#![feature(quote)]
+#![feature(quote, rustc_private)]
 
 extern crate syntax;
```

Comment:
I agree with @rprichard that these tests which link to `syntax` should be in compile-fail-fulldeps to ensure that the dependencies work out correctly during builds.